### ISSUE TITLE
Documentation for CVodeCreateB() is missing pointer

### ISSUE THREAD

**Issue by tepperly** [LLNL/sundials#323]
Looking here: https://sundials.readthedocs.io/en/latest/cvodes/Usage/ADJ.html#c.CVodeCreateB

The signature is missing the `*` before `which`. It should be:
```c++
int CVodeCreateB(void *cvode_mem, int lmmB, int *which)
```

**Comment by tepperly** [comment]
Similarly, https://sundials.readthedocs.io/en/latest/cvodes/Usage/ADJ.html#c.CVodeGetB , CVodeGetB is missing a pointer in front of `tret`. It should be:
```c++
int CVodeGetB(void *cvode_mem, int which, realtype *tret, N_Vector yB);
```

**Comment by tepperly** [comment]
Similarly, https://sundials.readthedocs.io/en/latest/cvodes/Usage/ADJ.html#c.CVodeF , CVodeF is missing the pointer in front of `ncheck`. It should be:
```c++
int CVodeF(void *cvode_mem, realtype tout, N_Vector yret, realtype *tret, int itask, int *ncheck)
```
```

**Comment by balos1** [comment]
Fixed in #325.
